flips/20220614-cadence-style-guide.md

```diff
@@ -0,0 +1,71 @@
+# Cadence Style Guide
+
+| Status        | (Proposed)       |
+:-------------- |:---------------------------------------------------- |
+| **FLIP #**    | [NNN](https://github.com/onflow/flow/pull/NNN) (update when you have PR #)|
+| **Author** | Álvaro Lillo Igualada (alvaro.lillo@dapperlabs.com) |
+| **Sponsor**   | Josh Hannan (josh.hannan@dapperlabs.com)             |
+| **Updated**   | 2022-06-23                                           | 
+
+## Objective
+
+To provide developers with better guidelines for how to write clean cadence code.
+
+## Motivation
+
+As the Cadence developer base extends, we need to unificate the coding style to make all Cadence code easy to read and save developers time writing it.
+
+
+## User Benefit
+
+Having unified rules when writting Cadence code will help Cadence learners understanding code and all kind of Cadence based projects not wasting time deciding how to present their code.
```

Comment:
```suggestion
Having unified standards when writting Cadence code will help Cadence learners understand code better. All kinds of Cadence-based projects will be able to avoid wasting time deciding how to present their code.
```